extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -275,11 +275,11 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * @private
    */
   cloneLightboxableElement_(element) {
-    const fallback = element.getFallback();
-    const shouldCloneFallback =
-      element.classList.contains('amp-notsupported') && !!fallback;
-    if (shouldCloneFallback) {
-      element = fallback;
+    if (element.classList.contains('amp-notsupported')) {
+      const fallback = element.getFallback();
+      if (!!fallback) {
```

Comment:
truthy/falsey should be fine here
```suggestion
      if (fallback) {
```